File: SortWare/Interfaces/YesNoApplyToAll.vb
Public Class YesNoApplyToAll

    Public Property applyAll As Boolean = False

    Public Sub New(ByVal msg As String)
        ' This call is required by the designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.
        MainLabel.Text = msg
    End Sub

    Private Sub YesButton_Click(sender As Object, e As EventArgs) Handles YesButton.Click
        Me.DialogResult = DialogResult.Yes
        applyAll = ApplyAllChkBx.Checked
    End Sub

    Private Sub NoButton_Click(sender As Object, e As EventArgs) Handles NoButton.Click
        Me.DialogResult = DialogResult.No
        applyAll = ApplyAllChkBx.Checked
    End Sub

    Private Sub CancelButton_Click(sender As Object, e As EventArgs) Handles CancelButton.Click
        Me.DialogResult = DialogResult.Cancel
        applyAll = ApplyAllChkBx.Checked
    End Sub
End Class

File: SortWare/Interfaces/YesNoApplyToAll.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace SortWare
{
    public partial class YesNoApplyToAll
    {

        public bool applyAll { get; set; } = false;

        public YesNoApplyToAll(string msg)
        {
            // This call is required by the designer.
            InitializeComponent();

            // Add any initialization after the InitializeComponent() call.
            MainLabel.Text = msg;
            MainLabel.MaximumSize = new Size(480, 0);
            MainLabel.AutoSize = true;
        }

        private void YesButton_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.Yes;
            applyAll = ApplyAllChkBx.Checked;
        }

        private void NoButton_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.No;
            applyAll = ApplyAllChkBx.Checked;
        }

        private void CancelButton_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.Cancel;
            applyAll = ApplyAllChkBx.Checked;
        }
    }
}